### PR TITLE
Successors select for all package Dialogs

### PR DESCRIPTION
**Pull request by fiskus**
"Push package to bucket" dialog is untouched, but I think it's even obsolete now
Screenshot from 2022-05-10 14-08-22
Screenshot from 2022-05-10 14-10-05

### CODE REVIEW

**Comment by nl0** [comment]
re: select styling: we already have an inline bucket select dropdown and it looks a little bit different:
Screenshot 2022-05-06 at 18 11 39

i think we should be consistent and use the same styling (either one, i don't care too much)

**Comment by fiskus** [comment]
The one I use is also used for selecting file "modes" (JSON/Jupyter/Voila) and to select search mode (objects/packages). I'd adopt it everywhere in the long term

**Comment by fiskus** [comment]
Changed to `variant="text"`

**Review by nl0 — APPROVED** [review]

File: catalog/app/containers/Bucket/Successors.tsx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+  error: Error
+}
+
+function ErrorSlot({ error }: ErrorSlotProps) {
+  return (
+    <M.Box px={2} py={1}>
+      <M.Typography gutterBottom>
+        Error: <code>{error.message}</code>
+      </M.Typography>
+      {R.is(ERRORS.WorkflowsConfigInvalid, error) && (
```
[the remark this answers is not in the document]
Comment (fiskus) [reply catalog/app/containers/Bucket/Successors.tsx:45]: There was some problem with the resulting code from the Typescript compiler, but it seems it's gone now. I'll update this code in another PR after more testing and reading

**Pull request merged**